### ISSUE TITLE
Difference in GCC build instructions for 20171104 and 20170703 versions

### ISSUE THREAD

**Issue by cherdt** [libyal/libewf#95]
On https://github.com/libyal/libewf/wiki/Building the usual build steps for GCC:

```
./configure
make
sudo make install
```

work for [libewf-experimental-20170703.tar.gz](https://github.com/libyal/libewf/releases/download/20170703/libewf-experimental-20170703.tar.gz) but not [libewf-experimental-20171104.tar.gz](https://github.com/libyal/libewf/releases/download/20171104/libewf-experimental-20171104.tar.gz).

The latter appears to use autoconf/automake instead (although I ran into errors with autoconf as well and was unable to build that version).

**Comment by joachimmetz** [comment]
How did you download libewf, via dist package or from git?

What is the error/issue you are encountering?

**Comment by cherdt** [comment]
I downloaded `libewf` from the [Releases](https://github.com/libyal/libewf/releases) page. I included the specific download links in my previous message.

Here's the error when trying to run `autoconf` for the 20171104 release on CentOS 7:

```
[root@localhost libewf-20171104]# autoconf
configure.ac:11: error: possibly undefined macro: AM_INIT_AUTOMAKE
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:19: error: possibly undefined macro: AC_LIBTOOL_WIN32_DLL
configure.ac:28: error: possibly undefined macro: AC_PROG_LIBTOOL
configure.ac:35: error: possibly undefined macro: AM_GNU_GETTEXT
configure.ac:36: error: possibly undefined macro: AM_GNU_GETTEXT_VERSION
configure.ac:140: error: possibly undefined macro: AM_CONDITIONAL```

Version of `autoconf`:

```
# autoconf -V
autoconf (GNU Autoconf) 2.69
```

**Comment by joachimmetz** [comment]
you don't need to run autoconf (autotools) when you use the dist package, just use the configure script that is distributed with the source

**Comment by cherdt** [comment]
There was no configure script included in the 20171104 dist package I downloaded. However, I just downloaded another copy of the dist package and it does contain the configure script.

I can't explain it, I must have done something odd that I can't recall. Sorry for the confusion!

**Comment by joachimmetz** [comment]
> There was no configure script included in the 20171104 dist package I downloaded.

then it's unlikely the dist package but a github generated tar.gz of the source. Alas this functionality cannot be disabled in the github UI as far as I know.
